src/hooks/usePermalink.ts

```diff
@@ -53,7 +53,7 @@ interface HookResult {
 export const usePermalink: (args: Args) => HookResult = ({ room, type: argType, url }): HookResult => {
     const [member, setMember] = useState<RoomMember | null>(null);
     // room of the entity this pill points to
-    const [targetRoom, setTargetRoom] = useState<Room | undefined | null>(room);
+    const [targetRoom, setTargetRoom] = useState<Room | null>(room || null);
```

Comment:
we should probably use `??` rather than `||`  as a matter of good practice? Not that it makes any difference here since a Room can never be falsey.